### DISCUSSION TITLE
How can I add different speed multiplicand and damage multiplicand to varying axes?

### DISCUSSION THREAD

**Discussion by Aasterza** [RakambdaOrg/FallingTree#518] [Q&A]
Howdy! Through the config I was trying to give different axes different cutdown speeds; an example being a broad axe would cut down a tree at 0 speed multiplicand and a battle axe would cut down a tree at 0.5 speed multiplicand. Is this something I'd have to do with datapacks? If so, is there a tutorial on how to do so?

**Comment by Rakambda** [comment]
This isn't currently possible. The breaking speed is tied to the mod itself, not a specific axe.
This can be a suggestion I can take into account. However I currently have no idea on how to do it in a simple way (doesn't need extra blocks and such as it'd break server-side only when not using breaking speed changes).
